### DISCUSSION TITLE
theme notebook cell borders in dark mode

### DISCUSSION THREAD

**Discussion by jcmgray** [pradyunsg/furo#336] [Ideas] [answered]
Thanks for the beautiful theme! 

I'd be keen to use this theme with either nbsphinx or myst_nb to render notebook docs - it already looks great in light mode. In dark mode the cells do have a pronounced white border that is a bit jarring however:

![Screenshot 2021-12-18 000424](https://user-images.githubusercontent.com/8982598/146621301-2cd13b98-03bc-4d1b-89d3-bfd9d4cf7144.png)

and here's maybe a desired look after tweaking some parameters (border and background color) in inspect element...

![Screenshot 2021-12-18 000642](https://user-images.githubusercontent.com/8982598/146621318-a1b4a33f-dacd-4ab3-ae2a-2a181f26cd52.png)

just wondering if this kind of special case element theming might make it into the theme?

**Comment by pradyunsg** [comment]
I'm not familiar with myst-nb or nbsphinx, but special-casing extensions is welcome -- there's already a few of those in https://github.com/pradyunsg/furo/tree/main/src/furo/assets/styles/extensions

**Reply by chrisjsewell** [reply]
For myst-nb, let me know if this requires any upstream improvements 😄 the CSS is pretty simple: https://github.com/executablebooks/MyST-NB/blob/master/myst_nb/static/mystnb.css

**Comment by pradyunsg** [comment, marked as answer]
A PR would be welcome for this.

The way I'm imagining this can be achieved is by adding a new file to https://github.com/pradyunsg/furo/tree/main/src/furo/assets/styles/extensions, setting the relevant CSS variables needed based on https://github.com/executablebooks/MyST-NB/blob/master/myst_nb/static/mystnb.css in `html` (instead of `:root`) based off of Furo's own internal CSS variables (similar to how sphinx-inline-tabs is supported).
